common-theurgy.lic

```diff
@@ -113,6 +113,7 @@ module DRCTH
   end
 
   def empty_cleric_hands(theurgy_supply_container)
+    fput('glance')
```

Comment:
Can you add a comment that this is because the item can change (with the example you have in the description)?